docs/subnets/walkthrough-prompting.md

```diff
@@ -0,0 +1,117 @@
+
+<center id="bittensor-img">
+<ThemedImage
+alt="1-Prompting Walkthrough"
+sources={{
+    light: useBaseUrl('/img/docs/1-prompting-subnet-walkthrough.svg'),
+    dark: useBaseUrl('/img/docs/1-prompting-subnet-walkthrough.svg'),
+  }}
+style={{width: 600}}
+/>
+</center>
+
+## Subnet 1 summary
+
+See the below diagram showing a high-level view of how this Prompting subnet works. 
+
+<center>
+<ThemedImage
+alt="Prompting Subnet 1 Big Picture"
+sources={{
+    light: useBaseUrl('/img/docs/2-prompting-subnet-high-level.svg'),
+    dark: useBaseUrl('/img/docs/2-prompting-subnet-high-level.svg'),
+  }}
+style={{width: 750}}
+/>
+</center>
+
+The numbered items in the below description correspond to the numbered sections in the above diagram:
+
+1. The subnet validator sends a **challenge** simultaneously to multiple subnet miners. This step constitutes the **prompt** from the subnet validator to subnet miners. A challenge is a prompt sent to subnet miners in such a way that:
+   - This prompt is in a style and tone that is similar to humans.
+   - This prompt drives the conversation between the subnet validator and the subnet miners in order to reach a pre-defined goal. 
+
+    See the below [Challenge generation](#challenge-generation) section for how a challenge is generated. 
+2. The subnet miners respond to the subnet validator after performing the challenge **task**. 
+3. The subnet validator then **scores** each subnet miner by comparing the subnet miner's response with a locally generated **reference** answer. The subnet validator uses this reference as the ground truth for this step. 
+4. Finally, the subnet validator **sets the weights** for the subnet miners by sending the weights to the blockchain. In the Bittensor blockchain the Yuma Consensus allocates the rewards to the participating subnet miners and subnet validators. 
+
+:::tip Use of large language models 
+In this subnet both the subnet validator and the subnet miners use large language models (LLMs) to create the challenges (subnet validator) and respond to the prompts (subnet miners).
+:::
+
+## Key innovations in this subnet
+
+This subnet has developed a few innovative techniques to get to a real human-like conversational AI that actually produces intelligence instead of copying from the internet. See below:
+
+### Achieving human-like conversation
+
+To deliver to a user of this subnet an experience of a human-like conversation:
+
+- Subnet validators perform a roleplay where they take on the persona of **random** human users before they prompt the subnet miners. By doing this, the subnet validators engage the subnet miners in a real, random, human-like conversation throughout the subnet operation.
+- This means that throughout the subnet validation process the subnet miners continually improve their performance as though all validation prompts originated from an external API client query. This enables the subnet miners to become better intelligent AI assistants. 
```

Comment:
```suggestion
- This means that throughout the subnet validation process the subnet miners continually improve their performance as though all validation prompts originated from an external API client query. This enables the subnet miners to become better intelligent AI assistants. 
```
Don’t love the way this reads. Maybe integrate my comments above regarding handling ‘fuzzy’ instructions